src/main/scala/DemoFortis.scala

```diff
@@ -173,6 +181,66 @@ object DemoFortis {
       }
     }
 
+    if (mode.contains("tadaweb")) {
+      streamProvider.buildStream[TadawebEvent](ssc, streamRegistry("tadaweb")) match {
+        case Some(stream) => stream
+          .map(event => AnalyzedItem(
+            event,
+            source = event.tada.name,
+            analysis = Analysis(keywords =
+              keywordExtractor.extractKeywords(event.text) ::: keywordExtractor.extractKeywords(event.title)
+            )
+          ))
+          .filter(_.analysis.keywords.nonEmpty)
+          .map(fortisEvent => {
+            val language = languageDetection.detectLanguage(fortisEvent.originalItem.text)
+            val sentiment: Option[Double] = fortisEvent.originalItem.sentiment match {
+              case "negative" => Some(0)
+              case "neutral" => Some(0.6)
+              case "positive" => Some(1)
+              case _ => language match {
+                case Some(lang) =>
+                  if (supportedLanguages.contains(lang))
```

Comment:
That said, I will also add a filter to prevent items without a sentiment from flowing onward.